.github/workflows/linux.yml

```diff
@@ -911,8 +922,7 @@ jobs:
           popd
 
       - name: Install OpenVINO Python wheels
-        run: |
-          python3 -m pip install openvino --find-links=${{ env.INSTALL_DIR }}/tools
+        run: python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
```

Comment:
openvino-dev should not be install this way because the name of the dev wheel is openvino**_**dev-2023.x.x,

You can check that in the logs:
Run python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
Processing ./install/tools/openvino-2023.2.0-1-cp311-cp311-manylinux_2_31_x86_64.whl
...
Installing collected packages: openvino-telemetry, numpy, openvino